modules/ROOT/partials/menu-item-ids/typography-menu-items.adoc

```diff
@@ -0,0 +1,5 @@
+[cols="1,1,2",options="header"]
+|===
+|Menu item identifier |xref:menus-configuration-options.adoc#example-the-tinymce-default-menu-items[Default Menu Location] |Description
+|`+typography+` |Custom | Provides two sub-menu items: Enhance, which runs the currently active set of Advanced Typography rules over the document or the current selections; and Language, which displays a further menu listing all the language-specific rule sets that can be selected.
```

Comment:
Should we mention that if you select a language in the menu, it will apply it also?